### DISCUSSION TITLE
HMIP Broll unreach after first button press

### DISCUSSION THREAD

**Discussion by atze09** [SukramJ/aiohomematic#1217] [General]
### I agree to the following

- [X] I have read the [documentation](https://github.com/danielperna84/custom_homematic#custom_homematic)
- [X] I am aware of the latest [release notes](https://github.com/danielperna84/custom_homematic/releases)
- [X] The Backend (CCU/Homegear/...) is working as expected and connected devices are controllable by its own UI.
- [X] I am running the latest version of the custom_component (and Home Assistant)

### The problem

Everything is working intially, but after i press one button my covers move, but get unreach/grey in HA after the first button press.
In Raspberrymatic i see that the HMIP Broll are unreach but i can control them in Raspberymatic although they are under the state unreach. But in HA they are grey-.

### What version of HomematicIP (local) has the issue?

1.32.1

### What type of installation are you running?

Home Assistant OS

### What type of installation are you running for your homematic backend?

RaspberryMatic HA-Addon

### Which version of your homematic backend are you running?

3.71

### What hardware are you running for your system?

Yellow

### Which config details do you use

- [X] Username and password
- [X] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [X] Homematic IP
- [X] Homematic (Bidcos-RF)
- [ ] Groups (Heating-Group)
- [ ] BidCos-Wired (HM-Wired)

### Diagnostics information

[home-assistant_homematicip_local_2023-09-29T19-14-02.099Z.log](https://github.com/danielperna84/hahomematic/files/12770812/home-assistant_homematicip_local_2023-09-29T19-14-02.099Z.log)

**Comment by Lemocin** [comment]
Hi, sounds like a communication issue. Try to use this: https://github.com/danielperna84/custom_homematic#homematicip_localforce_device_availability

**Comment by atze09** [comment]
> Hi, sounds like a communication issue. Try to use this: https://github.com/danielperna84/custom_homematic#homematicip_localforce_device_availability

I tried it and it did not work, before under the CCU3 evrything was working fine.

**Comment by SukramJ** [comment]
I agree with what @Lemocin said.
Since I don't think there's a problem with the integration and I also refuse to support a 6 month old versions, I'm moving this issue into a discussion.

**Reply by SukramJ** [reply]
@atze09 
Wenn du, nachdem Du auf eine aktualisierte Version aktualiert hast hier Informationen einstellst, wären  die `Diagnostics information` hilfreich (Klick auf Integration --> Diagnose Information herunterladen, Anleitung stand auch im Issue).
Wie dem Log zu entnehmen ist stimmt was nicht mit deiner TLS Konfiguration, also TLS deaktivieren, oder richtig konfigurieren (auch CCU).
